--- a/hostapd/main.c
+++ b/hostapd/main.c
@@ -741,7 +741,8 @@ static void show_version(void)
 		"hostapd v%s\n"
 		"User space daemon for IEEE 802.11 AP management,\n"
 		"IEEE 802.1X/WPA/WPA2/EAP/RADIUS Authenticator\n"
-		"Copyright (c) 2002-2024, Jouni Malinen <j@w1.fi> "
+		"*** WLAN Pi Profiler Custom Build - DFS DISABLED - TESTING ONLY ***\n"
+		"Upstream Copyright (c) 2002-2024, Jouni Malinen <j@w1.fi> "
 		"and contributors\n",
 		VERSION_STR);
 }
